From 5aa37e4da53e537bc2e073f2e317fd7ef1933d14 Mon Sep 17 00:00:00 2001
From: Valentine Barshak <vbarshak@dev.rtsoft.ru>
Date: Sat, 29 Dec 2012 22:16:21 +0400
Subject: [PATCH 020/127] arm: tegra: ktt30: Enable watchdog on probe

This adds CONFIG_TEGRA_WATCHDOG_ENABLE_ON_PROBE=y to ktt30_defconfig.

Signed-off-by: Valentine Barshak <vbarshak@dev.rtsoft.ru>
---
 arch/arm/configs/ktt30_defconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/configs/ktt30_defconfig b/arch/arm/configs/ktt30_defconfig
index 4ac4529..30da6fd 100644
--- a/arch/arm/configs/ktt30_defconfig
+++ b/arch/arm/configs/ktt30_defconfig
@@ -276,6 +276,7 @@ CONFIG_SENSORS_INA219=y
 CONFIG_THERMAL=y
 CONFIG_WATCHDOG=y
 CONFIG_TEGRA_WATCHDOG=y
+CONFIG_TEGRA_WATCHDOG_ENABLE_ON_PROBE=y
 CONFIG_MFD_TPS6591X=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
-- 
1.7.4.4

